integration-tests/src/test/resources/features/error-handling.feature

```diff
@@ -0,0 +1,52 @@
+Feature: Error handling tests
+
+  @errorhandling
+  Scenario: Error handling Webhook is correctly called if an error occurs
+    Given authenticate against Manager
+    And create a new Bridge with body:
+    """
+    {
+        "name": "ehBridge",
+        "errorHandler": {
+            "type": "Webhook",
+            "parameters": {
+                "endpoint": "https://webhook.site/${env.webhook.site.uuid}"
+            }
+        }
+    }
+    """
+    And the Bridge "ehBridge" is existing with status "ready" within 4 minutes
+    And the Ingress of Bridge "ehBridge" is available within 2 minutes
+
+    And add a Processor to the Bridge "ehBridge" with body:
+    """
+    {
+        "name": "whProcessor",
+        "action": {
+            "type": "Webhook",
+            "parameters": {
+                "endpoint": "https://example.com/dummy-webhook"
+            }
+        },
+        "transformationTemplate": "{\"sender_name\":\"{data.sender.name}\"}"
+    }
+    """
+    And the list of Processor instances of the Bridge "ehBridge" is containing the Processor "whProcessor"
+    And the Processor "whProcessor" of the Bridge "ehBridge" is existing with status "ready" within 3 minutes
+    And the Processor "whProcessor" of the Bridge "ehBridge" has action of type "Webhook" and parameters:
+      | endpoint | https://example.com/dummy-webhook |
+
+    When send a cloud event to the Ingress of the Bridge "ehBridge":
+    """
+    {
+        "specversion": "1.0",
+        "type": "myType",
+        "source": "mySource",
+        "id": "error-handling-test",
+        "subject": "mySubject",
+        "data": {
+            "myMessage": "Hello world!"
+        }
+    }
+    """
+    Then Webhook site contains request with text ""id":"${cloud-event.error-handling-test.id}","source":"mySource","type":"myType","subject":"mySubject"" within 1 minute
```

Comment:
I know it seems wrong, but it actually works because it matches the regex defined in [this line of WebhookSiteSteps class](https://github.com/kostola/sandbox/blob/973abac7384f1011dcfbaad255241f6691f17bf9/integration-tests/src/test/java/com/redhat/service/smartevents/integration/tests/steps/WebhookSiteSteps.java#L29)